framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -1491,4 +1491,22 @@ protected static void insertIntoStores(
       result.getRegularStore().insertValue(target, newAnno);
     }
   }
+
+  /**
+   * Inserts {@code newAnno} into all stores (conditional or not) in the result for node, while
+   * permitting non-determinism. This is a utility method for subclasses.
+   *
+   * @param result the {@link TransferResult} holding the stores to modify
+   * @param target the receiver whose values should be modified
```

Comment:
If this works for any JavaExpression (not just receivers), then please adjust the documentation to so note.